### ISSUE TITLE
How to get more then one currency?

### ISSUE THREAD

**Issue by Flippo24** [swimclan/gdax-candles#2]
I am trying to get two currencies, but ich only get candles for one.
What i am doing wrong?

```
const Chart = require('./src/Chart');

const product = 'ETH-USD';
const product2 = 'BTC-USD';
const timeframe = '1m';
const ethereumChart = new Chart({ product, timeframe }).start();
const bitcoinChart = new Chart({ product2, timeframe }).start();

ethereumChart.on('close', candle => {
  console.log(candle);
});
bitcoinChart.on('close', candle => {
  console.log(candle);
});
```

and got:

```
Candlestick {
  timestamp: 2018-07-04T14:07:55.024Z,
  open: 474.74,
  price: 475.09,
  close: 475.09,
  high: 475.51,
  low: 474.74,
  closed: true,
  height: 0.000736702519522545,
  spread: 0.0016207455429497186,
  volume: 22.706467770000007,
  sma: {},
  ema: {},
  color: 'green',
  regression: {} }
Candlestick {
  timestamp: 2018-07-04T14:07:55.024Z,
  open: 474.74,
  price: 475.09,
  close: 475.09,
  high: 475.51,
  low: 474.74,
  closed: true,
  height: 0.000736702519522545,
  spread: 0.0016207455429497186,
  volume: 22.706467770000007,
  sma: {},
  ema: {},
  color: 'green',
  regression: {} }
```

**Comment by swimclan** [comment]
You arent doing anything wrong.  For some reason (I cant remember why) I implemented the exchange class as a singleton.  This means that every time you create a chart instance, it will get the singleton exchange instance which is whatever you instantiated first. 

So....

That said, you bring up a good point.  This needs to NOT be a singleton so that you can spin up multiple charts.  I will consider this a bug and do an update that should not be a breaking change.

Thanks for the feedback.  I should have an update done before week's end.

**Comment by swimclan** [comment]
@Flippo24 fixed in v1.5.0.  Please `npm update gdax-candles` in your project.

**Comment by Flippo24** [comment]
Thanks for your very fast help. This works like charm.
